### ISSUE TITLE
bindings and bindingOverrides doesn't work with neovim

### ISSUE THREAD

**Issue by sardnar** [VSpaceCode/VSpaceCode#361]
## Bug description

I am trying to customize vspacecode but it seems there is a conflict with neovim somehow. 

## To Reproduce
1. have neovim install and adjust config accordingly #34 
2. Adjust settings.json by copying the examples from https://vspacecode.github.io/docs/menu-customization either for bindingOverrides or bindings. 

## Expected behavior
This should have created new menu if a menu does not exit or override the entire menu.

## Screenshots
![image](https://github.com/VSpaceCode/VSpaceCode/assets/133683253/267495b0-f430-45a2-abdd-45489a0559a0)

I have restarted the entire vscode

## System information

Windows 10

**Comment by stevenguh** [comment]
Have you figured it out? I don't see any issues in the your settings in the screenshot although I've seen issue due to typo.

**Comment by krusse** [comment]
I also have this problem and have found out more about the possible cause.

**TLDR: it seems that the keybinding for vspacecode.space must exist for the bindingOverrides to be recognized.**

I tried to move the keybinding of space into the init.vim instead of vscode keybindings.json. The reason was that I often change text in vim with the motion ct<space> (i.e., change to next space).

When the recommended keybindings.json config is in place, 
 ```
 {
    "key": "space",
    "command": "vspacecode.space",
    "when": "editorTextFocus && neovim.mode == normal"
  },
```
the change to motion is interrupted by the which.key menu popping up instead of changing to next space (and my bindingOverrides are there).


When I use the keymap in init.vim:
```
  nnoremap <space> <Cmd>lua require('vscode').action('whichkey.show')<CR>
```
ct<space> works and <space> alone will open up the which.key menu, but my bindingOverrides are lost.

**Comment by stevenguh** [comment]
Yeah vscodecode uses whichkey and preserve both commands, but I can see the confusion. (At this point, it is mostly a historical artifact).

Basically if you use vspacecode, you should use `vspacecode.space`.
